### PR TITLE
Mepts 1055 | Changes MQ 15 Age

### PR DESCRIPTION
**Pull request by eliazmuchave**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/EptsCommonDimension.java [hunk 1]
```diff
@@ -132,6 +132,9 @@ public CohortDefinitionDimension age(AgeDimensionCohortInterface ageDimensionCoh
     dim.addCohortDefinition(
         DimensionKeyForAge.bellow2Years.getKey(),
         ageDimensionCohort.createXtoYAgeCohort("patients with age below 2 years", 0, 1));
+    dim.addCohortDefinition(
+        DimensionKeyForAge.overOrEqualTo2Years.getKey(),
+        ageDimensionCohort.createXtoYAgeCohort("patients with age below 2 years", 2, null));
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/EptsCommonDimension.java:137]: Sounds like it should be named **... over or equal to 2 years**
Please correct the name of the cohort

Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/EptsCommonDimension.java]: Sounds like it should be named **... over or equal to 2 years**
Please correct the name of the cohort

Comment (eliazmuchave) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/EptsCommonDimension.java]: Fixed

**Review by helderjosue — APPROVED** [review]
Looks good to me!

**Review by ningosi — APPROVED** [review]

**Pull request merged**